sstables_loader.cc

```diff
@@ -454,6 +454,8 @@ class sstables_loader::download_task_impl : public tasks::task_manager::task::im
     sstring _bucket;
     sstring _ks;
     sstring _cf;
+    sstring _prefix;
+    std::vector<sstring> _sstables;
     sstring _snapshot_name;
```

Comment:
nit: It's now unused. Can be fixed as a followup

Reply:
sure. will do.

Reply:
#20969